.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# Summary
+## Closes
+<!-- Some issue -->
+
+## Changes
+<!-- Ipsum lorem. -->
+ 
+## Specific Testing (Optional)
+<!-- Any specific text used to test outside of typical Ruff workflows -->
```

Comment:
To fix `pre-commit`.
```suggestion
## Closes

<!-- Some issue -->

## Changes

<!-- Ipsum lorem. -->
 
## Specific Testing (Optional)

<!-- Any specific text used to test outside of typical Ruff workflows -->
```

Reply:
Thanks for your help - sorry for the duplicate commit suggestions, have been away from my computer.